doc/news/changes.h

```diff
@@ -269,8 +269,9 @@ inconvenience this causes.
  (Bruno Turcksin, 2016/05/22)
   </li>
 
- <li> Move FESystem's auxiliary functions to FETools and extend them to cover
- a non-tensor product case used in the enriched finite element.
+ <li> New: There are now additional functions in namespace FETools that build
+ finite elements out of simpler finite elements, either by forming tensor
+ product or by enriching the set of shape functions.
```

Comment:
> by enriching the set of shape functions.

this is not done in those functions, so i would say it not entirely correct to say so. It will be later used to create enriched finite element, but the functions themselves are agnostic to the usage.